ui/.template-lintrc.js

```diff
@@ -1,45 +1,8 @@
-/**
- * Copyright (c) HashiCorp, Inc.
- * SPDX-License-Identifier: BUSL-1.1
- */
-
-'use strict';
-
-const fs = require('fs');
-let testOverrides = {};
-try {
-  // ember-template-lint no longer exports anything so we cannot access the rule definitions conventionally
-  // read file, convert to json string and parse
-  const toJSON = (str) => {
-    return JSON.parse(
-      str
-        .slice(str.indexOf(':') + 2) // get rid of export statement
-        .slice(0, -(str.length - str.lastIndexOf(','))) // remove trailing brackets from export
-        .replace(/:.*,/g, `: ${false},`) // convert values to false
-        .replace(/,([^,]*)$/, '$1') // remove last comma
-        .replace(/'/g, '"') // convert to double quotes
-        .replace(/(\w[^"].*[^"]):/g, '"$1":') // wrap quotes around single word keys
-        .trim()
-    );
-  };
-  const recommended = toJSON(
-    fs.readFileSync('node_modules/ember-template-lint/lib/config/recommended.js').toString()
-  );
-  const stylistic = toJSON(
-    fs.readFileSync('node_modules/ember-template-lint/lib/config/stylistic.js').toString()
-  );
-  testOverrides = {
-    ...recommended,
-    ...stylistic,
-    prettier: false,
-  };
-} catch (error) {
-  console.log(error); // eslint-disable-line
-}
```

Comment:
A change in the `ember-template-lint` files broke this hack during the last upgrade but there is a better way to do this. I briefly tried converting this to `mjs` and importing the rules but it didn't seem to work, although it should. I think we can circle back around on that though and get the rules overridden in the tests in VSCode since the Ember language server plugin doesn't respect the `ignore` field.